src/components/charts/colorMixingTool/index.jsx

```diff
@@ -232,24 +227,35 @@ class ColorTool extends React.PureComponent {
                 </div>
               );
             })}
-          {panelIsActive && (
-            <Button
-              raised
-              primary
-              disabled={!resetBtnActive}
-              className={resetBtn}
-              onClick={this.handleReset}
-            >
-              Reset
-            </Button>
-          )}
+          <Button
+            raised
+            primary
+            style={{ visibility: `${visibilityVal}` }}
+            disabled={!resetBtnActive}
+            className={resetBtn}
+            onClick={this.handleReset}
+          >
+            Reset
+          </Button>
         </div>
+        <Button
+          floating
+          className={panelClassName}
+          onClick={this.handleControlPanel}
+        >
+          {panelIsActive ? 'Off' : 'On'}
+        </Button>
```

Comment:
some css to help center your text:
```
padding: 0;
text-align: center;
```
You can apply these directly to the element via `style={{}}` or use a class. I would most likely suggest a class in this instance.